docs/_navbar.md

```diff
@@ -4,3 +4,4 @@
   - [:kr: ko-kr](/ko-kr/)
   - [:brazil: pt-br](/pt-br/)
   - [:czech_republic: cs](/cs/)
+  - [:es: spa](/spa/)
```

Comment:
You should probably use ISO 639-1 code for Spanish, which is `es`. (https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)

Reply:
Oh ok, fixing it

Reply:
I think a lot of the tools use these two letters, optionally you can also use another two letter for a country, like `en` (english) with `us` (USA) has. So something like `es-sp` or `es-mx`? See http://www.lingoes.net/en/translator/langcode.htm

Reply:
There is a problem with that, there no is official Spanish but we often use a standard but this standard does not belong to any country, should I put my country then? In my case is the Dominican Republic

Reply:
I would wait for a suggestion from @felangel. Don't know what opinion he has on that. ;)

Reply:
All right and thanks for the feedback 👍

Reply:
I would recommend leaving it as es for now and if we need to add country codes later we can 👍

Reply:
Sounds good to me